src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ValidationErrors.feature

```diff
@@ -0,0 +1,60 @@
+Feature: ValidationErrors
+    POST a request that has an invalid payload.
+
+Scenario: Post an empty request object
+ When sending a POST request to "data/ed-fi/schools" with body
+  """
+  """
+ Then the response code is 400
+    And the response body is
+"""
+{"detail":"Data validation failed. See errors for details.","type":"urn:dms:bad-request:data","title":"Data Validation Error","status":400,"correlationId":null,"validationErrors":{},"errors":["A non-empty request body is required."]}
+"""
+
+Scenario: Post an invalid body for academicWeeks when weekIdentifier length should be at least 5 characters
+ When sending a POST request to "data/ed-fi/academicWeeks" with body
+ """
+ {
+  "weekIdentifier": "one",
+  "schoolReference": {
+    "schoolId": 17012391
+  },
+  "beginDate": "2023-09-11",
+  "endDate": "2023-09-11",
+  "totalInstructionalDays": 300
+ }
+ """ 
+ 	Then the response code is 400
+    And the response body is
+"""
+{"detail":"Data validation failed. See validationErrors for details.","type":"urn:dms:bad-request:data","title":"Data Validation Error","status":400,"correlationId":null,"validationErrors":{"weekIdentifier : ":["weekIdentifier : Value should be at least 5 characters"]},"errors":[]}
+"""
```

Comment:
Can you change the `detail` to put apostrophes around `'validationErrors'`? Do we have enough information to also add the maximum length? ODS/API reports the error as `"weekIdentifier must be between 5 and 80 characters in length."` It would be good to match that if possible.

```json
{
  "detail": "Data validation failed. See 'validationErrors' for details.",
  "type": "urn:ed-fi:api:bad-request:data",
  "title": "Data Validation Failed",
  "status": 400,
  "correlationId": "01afa2ea-7a8e-48db-8659-3c5b26679281",
  "validationErrors": {
    "$.weekIdentifier": [
      "WeekIdentifier must be between 5 and 80 characters in length."
    ]
  }
}
```

If adding the max length is more than 5 minutes of work, then we can defer that to a separate ticket.

Reply:
The error is being returned to me in this way by the validator, and I think it would require a little more time to get to that level of detail.